modules/shared/attachments/redpanda-console-config.yaml

```diff
@@ -1,430 +1,325 @@
-    # SCRAM-SHA-512, GSSAPI, OAUTHBEARER and AWS_MSK_IAM.
-    #mechanism: PLAIN
-    # Kerberos implements the GSSAPI protocol. Use the following configurations
-    # to connect Redpanda Console to a Kerberos-protected Kafka API.
-    #gssapi:
-      # AuthType must be either USER_AUTH or KEYTAB_AUTH
-      #authType:
-      # Filepath to the keytab file that contains the pairs of Kerberos
-      # principals and encrypted password keys. AuthType must be set to
-      # KEYTAB_AUTH to load the keytab file.
-      #keyTabPath:
-      # Path to the krb5.conf file that contains Kerberos configuration
-      # information, such as the locations of KDCs.
-      #kerberosConfigPath:
-      #serviceName:
-      #username:
-      # Password can also be set using the --kafka.sasl.gssapi.password flag.
-      #password:
-      #realm:
-      # Enables FAST, which is a pre-authentication framework
-      # for Kerberos.
-      #enableFast: true
-    #oauth:
-      # Token can also be set using the --kafka.sasl.oauth.token flag.
-      #token:
-    #awsMskIam:
-      #accessKey:
-      # secretKey can also be set using the --kafka.sasl.aws-msk-iam.secret-key flag.
-      #secretKey:
-      # sessionToken can also be set using the --kafka.sasl.aws-msk-iam.session-token flag.
-      #sessionToken:
-      #userAgent:
-  #tls:
-  # If you connect to a Cluster that uses commonly trusted
-  # certificates, enable TLS and do not provide a certificate
-  # authority in the caFilepath. In this case,
-  # the system's certificate pool is used.
-    #enabled: false
-    #caFilepath:
-    #certFilepath:
-    #keyFilepath:
-    #insecureSkipTlsVerify: false
-  #schemaRegistry:
-    #enabled: false
-    # Url with scheme is required, for example ["http://localhost:8081"].
-    #urls: []
-    # Basic auth username.
-    #username:
-    # Basic auth password. This can also be set using the --schema.registry.password flag.
-    #password:
-    # bearerToken can also be set using the --schema.registry.token flag.
-    #bearerToken:
-    #tls:
-    # If you connect to a schema registry that uses commonly trusted
-    # certificates, enable TLS and do not provide a certificate
-    # authority in the caFilepath. In this case,
-    # the system's cert pool is used.
-      #enabled: false
-      # Path to a custom CA file. If not specified the system/trusted root CA is used.
-      #caFilepath:
-      #certFilepath:
-      # Key should not be encrypted by a passphrase.
-      #keyFilepath:
-      #insecureSkipTlsVerify: false
-  #protobuf:
-    #enabled: false
-    #mappings: []
+    - "broker-0.mycompany.com:19092"
+    - "broker-1.mycompany.com:19092"
+    - "broker-2.mycompany.com:19092"
+  # Optional: Client ID used to identify Console to the Kafka cluster.
+  # clientId: "console"
+  # Optional: Rack identifier to optimize message consumption in multi-zone clusters.
+  # rackId: "zone-a"
+  # sasl:
+    # enabled: true
+    # Supported mechanisms include:
```

Comment:
Is PLAIN supported?

Reply:
@weeco did we add support for PLAIN?

Reply:
Has always been supported there, but we don't support PLAIN in the UI. As far as I know we do not yet support SASL_PLAIN

Reply:
I do not see it in the mechanism dropdown of the latest beta. I'll merge as-is and we can update before GA if necessary.